ES/ESmy016.xml

```diff
@@ -31,325 +31,126 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <objectDesc form="Codex">
                      <supportDesc>
                         <support>
-                           <material key="parchment"/>
+                           <material key="parchment">Parchment leaves are poorly cut, with uneven edges.
+                           Cuts or holes are crudely amended on <locus target="#34 #35 #44"/>.</material>
                         </support>
-                        <extent>Made of <measure unit="leaf">44.0</measure> folios, of which
-                              <measure unit="leaf" type="blank"/> blank 42rv, 44v, in <measure unit="quire">6.0</measure> quires . <dimensions type="outer">
+                        <extent>
+                          <measure unit="leaf">44</measure>
+                        <measure unit="leaf" type="blank">3</measure>
+                        <locus target="#42r #42v #44v"/>
+                        <measure unit="quire">6</measure>
+                        <dimensions type="outer">
                               <height unit="mm">145</height>
                               <width unit="mm">110</width>
                               <depth unit="mm">30</depth>
                            </dimensions>
                         </extent>
-                        <foliation><!--2016-10-18+02:00 Updated MY-016 from DomLib--></foliation>
+                        <foliation>Foliation by Ethio-SPaRE, in pencil, in the outer corner of the bottom margin.</foliation>
                         <collation>
                            <list>
-                              <item xml:id="q1">
-                                 <dim unit="leaf">7</dim>
-                                 <locus from="1r" to="8v"/>
+                              <item xml:id="q1" n="1">
+                                 <dim unit="leaf">8</dim>
+                                  <locus from="1r" to="8v"/>
+                                  </item>
+
+                                 <item xml:id="q2" n="2">
+                                   <dim unit="leaf">8</dim>
                                  <locus from="9r" to="16v"/>
-                                 <locus from="17r" to="24v"/>
+                                </item>
+
+                                 <item xml:id="q3" n="3">
+                                   <dim unit="leaf">8</dim>
+                                    <locus from="17r" to="24v"/>
+                                    </item>
+
+                                <item xml:id="q4" n="4">
                                  <locus from="25r" to="29v"/>
+                                 </item>
+
+                                 <item xml:id="q5" n="5">
+                                     <dim unit="leaf">8</dim>
                                  <locus from="30r" to="37v"/>
+                                 </item>
+
+                                 <item xml:id="q6" n="6">
+                                   <dim unit="leaf">7</dim>
```

Comment:
`<dim unit="leaf">7</dim>`VI(7/fols. 38r-44v; s.l.: ???, stub after ???)

the system will break unless you explain why the number of folia is uneven

Reply:
The structure of this quire is difficult to understand.

Reply:
The app won't visualize the manuscript at all if you leave an uneven number unexplained, you should make a guess and add a note that it is just a guess

Reply:
I did. Like this?

Reply:
No, please follow precisely the Ethio-SPaRe syntax, as this is the one the app reads, with "s.l."
s.l.: 1, no stub 
s.l.: 1, stub after 7

Reply:
I did now with s.l.:, is it what you meant? If it is, I should point to an inconsistency in the "guidelines", where s.l. (I also thought that s.l. should be).

Reply:
sorry you are right it works also without s.l., I just checked, but I think with s.l. it is clearer